### DISCUSSION TITLE
chrome extension有计划支持网页内容导入吗？

### DISCUSSION THREAD

**Discussion by Geekiter** [Future-Scholars/paperlib#186] [Ideas]
我看zotero是有的，还挺方便，能备份网页内容

**Comment by GeoffreyChen777** [comment]
目前感知不到为何要保存网页。对于论文来说，内容应当都在PDF里。如果是想存一些不是论文的东西，这不是paperlib设计的目的抱歉

**Reply by Geekiter** [reply]
嗯嗯，理解。或许我可以自己加点代码实现自己的需求。
